datafn/python/datafn/handlers/sync.py

```
@@ -0,0 +1,269 @@
+                    "errors": []
+                }
+                await store_idempotency(config.db, "datafn", m_client_id, m_id, result)
+                
+                applied.append(m_id)
+                
+            except DatafnError as e:
+                errors.append({
+                    "mutationId": m_id,
+                    "error": {"code": e.code, "message": e.message, "details": e.details}
+                })
+            except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)